### PR TITLE
Convert Meijer G-function and better support for Singular Points (regular).

### PR DESCRIPTION
**Pull request by shubhamtibra**

### CODE REVIEW

File: sympy/holonomic/holonomic.py [hunk 1]
```diff
@@ -1178,6 +1178,156 @@ def to_sequence(self, lb=True):
 
         return HolonomicSequence(sol, u0)
 
+    def _fobrenius(self, indicialroots, lb=True):
```
Comment (jksuom) [inline sympy/holonomic/holonomic.py]: The name is spelled 'Frobenius'.

**Comment by jksuom** [comment]
It seems that only real roots are sorted. What should be done with complex roots?

File: sympy/holonomic/holonomic.py [hunk 2]
```diff
@@ -1215,9 +1215,17 @@ def _frobenius(self, lb=True):
...
+        compl = []
         for i in indicialroots:
             if i.is_real:
                 reals.extend([i] * indicialroots[i])
+            else:
+                a, b = i.as_real_imag()
+                compl.extend([(i, a, b)] * indicialroots[i])
+        compl.sort(key=lambda x : x[1])
+        for i in range(0, len(compl)-1):
+            if compl[i][2] > 0 and compl[i][1] == compl[i+1][1]:
```
Comment (jksuom) [inline sympy/holonomic/holonomic.py]: There may be several roots having the same real part. If the coefficients of the indicial equation are not real, the roots are not necessarily conjugates of each other.

Comment (shubhamtibra) [inline sympy/holonomic/holonomic.py]: That is correct. I have removed this assumption.

**Comment by shubhamtibra** [comment]
Ping @jksuom @certik .

**Comment by certik** [comment]
That looks good to me. Great work.

**Pull request merged**